site/pt/blog/new-in-devtools-109/index.md

```diff
@@ -118,45 +146,57 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
 {% endAside %}
 
 <!-- The current **Breakpoints** pane provides little visual aid in overseeing all breakpoints. On top of that, frequently used actions are hidden behind the context menu. -->
+O painel **Breakpoints** atual fornece pouca ajuda visual para ver todos os pontos de interrupção. Além disso, as ações usadas com frequência estão ocultas atrás do menu de contexto.
 
 <!-- This experimental UX redesign aims at bringing structure into the **Breakpoints** pane and allow developers to have quick access to commonly used features, in particular editing and removing breakpoints. -->
+Este redesenho experimental de UX visa trazer estrutura para o painel **Breakpoints** e permitir que os desenvolvedores tenham acesso rápido aos recursos comumente usados, em particular edição e remoção de pontos de interrupção.
 
 <!-- These are some highlights: -->
-
+Estes são alguns destaques:
 <!-- - Both pause options are in the **Breakpoints** pane and labeled with text to make it more self-explanatory. -->
 <!-- - Breakpoints are grouped by file, ordered by line/column number, and collapsible.** -->
 <!-- - New options to remove and edit breakpoint when hovering over a breakpoint or file name in the **Breakpoint** pane. -->
+- Ambas as opções de pausa estão no painel **Breakpoints** e rotuladas com texto para torná-lo mais auto-explicativo.
+- Os breakpoints são agrupados por arquivo, ordenados por número de linha/coluna e recolhíveis.**
+- Novas opções para remover e editar o breakpoint ao passar o mouse sobre um deles ou no nome do arquivo no painel **Breakpoint**.
 
 <!-- Read the full changes in our [RFC (closed)](https://github.com/ChromeDevTools/rfcs/discussions/3) and leave your feedback [here](https://crbug.com/1394686). -->
+Leia as alterações completas em nosso [RFC (fechado)](https://github.com/ChromeDevTools/rfcs/discussions/3) e deixe seu feedback [aqui](https://crbug.com/1394686).
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ytfyl8qK5rkHQRTS3sXf.png", alt="Show Breakpoint pane before and after the redesign.", width="800", height="684" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ytfyl8qK5rkHQRTS3sXf.png", alt="Mostrar o painel Breakpoint antes e depois da reformulação.", width="800", height="684" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/f2140378e0bb1687b263c226de01b741487ff324 #}
-Chromium issues: [1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904) 
+Issues relacionadas: [1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904) 
 
 
 <!-- ## [Experimental] Automatic in-place pretty print -->
+## [Experimental] Formatação automática (pretty print) no mesmo painel
 
 {% Aside %}
 <!-- To enable the experiment, check **Automatically pretty print in the Sources panel** under **Settings** > **Experiments**. -->
+Para ativar o experimento, marque **Formatar impressão automaticamente no painel Fontes** em **Configurações** > **Experimentos**.
```

Comment:
Os experimentos não são traduzidos no devtool em portugues.
Para ativar o experimento, marque **Automatically pretty print in the Sources panel** em **Configurações** > **Experimentos**

Reply:
```suggestion
Para ativar o experimento, marque **Automatically pretty print in the Sources** em **Configurações** > **Experimentos**.
```